assets/js/components/wp-dashboard/WPDashboardPopularPages.stories.js

```diff
@@ -0,0 +1,61 @@
+/**
+ * Site Kit by Google, Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Internal dependencies
+ */
+import { withWidgetComponentProps } from '../../googlesitekit/widgets/util';
+import {
+	setupAnalyticsGatheringData,
+	setupSearchConsoleAnalyticsMockReports,
+	setupSearchConsoleAnalyticsZeroData,
+	widgetDecorators,
+} from './common.stories';
+import WithRegistrySetup from '../../../../tests/js/WithRegistrySetup';
+import WPDashboardPopularPages from './WPDashboardPopularPages';
+
+const WidgetWithComponentProps = withWidgetComponentProps( 'widget-slug' )(
+	WPDashboardPopularPages
+);
+
+const Template = ( { setupRegistry = () => {}, ...args } ) => (
+	<WithRegistrySetup func={ setupRegistry }>
+		<WidgetWithComponentProps { ...args } />
+	</WithRegistrySetup>
+);
+
+export const Ready = Template.bind( {} );
+Ready.storyName = 'Ready';
+Ready.args = {
+	setupRegistry: setupSearchConsoleAnalyticsMockReports,
```

Comment:
This component doesn't use data from Search Console so it isn't clear whether this is intentional or not. Can we use something specific for GA here instead?

Reply:
Good point. Updated.